src/Renci.SshNet/Renci.SshNet.csproj

```diff
@@ -33,7 +34,7 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'net462' or '$(TargetFramework)' == 'netstandard2.0' ">
-    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="1.0.0" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="8.0.0" />
```

Comment:
Can you revert this. You can check a history of this line.

Reply:
I reverted it back to version 1.0.0 and added a not so we don't try to update it again.